pkg/hooks/loader.go

```diff
@@ -174,6 +190,7 @@ func (h *Hook) SetTcsMocks(m []*models.Mock) {
 func (h *Hook) SetConfigMocks(m []*models.Mock) {
 	h.configMocks.deleteAll()
 	for index, mock := range m {
+		h.persistentMatchedMocks[mock.Name] = false
 		mock.TestModeInfo.SortOrder = index
```

Comment:
for sake of consistency UpdatePersistentMatchedMock

Reply:
done!